include/boost/iterator/filter_iterator.hpp

```diff
@@ -13,6 +13,13 @@
 #include <boost/type_traits/is_class.hpp>
 #include <boost/static_assert.hpp>
 
+#if __cplusplus >= 201103L
+#include <utility>
+#define STD_MOVE(x) std::move(x)
```

Comment:
No, this won't do. We don't define macros not qualified with a library prefix and since this is an internal macro, it should include `DETAIL` (e.g. `BOOST_ITERATOR_DETAIL_MOVE` or something) and preferably undefined at the end of the file. And the check for `__cplusplus >= 201103L` is also not correct because not all compilers set `__cplusplus` accordingly. You should use `BOOST_NO_CXX11_RVALUE_REFERENCES` defined by Boost.Config, and preferably avoid using `std::move` and including `<utility>`.